docs/developers/dependencies.md

```diff
@@ -0,0 +1,55 @@
+
+---
+title: Dependency Management
+---
+
+You may want your plugin to hook into another. Reinventing the wheel takes too long. Either way, dependencies are common,
+but they need to be managed wisely, lest you fall into dependency hell.
```

Comment:
Maybe we could write this better? I feel it doesn't really hook as well as it could